faas

```diff
@@ -134,6 +134,7 @@ function apply() {
 function delete() {
   echo "Deleting $FUNCTION resource[s]"
   kubectl delete function $FUNCTION
+  kubectl delete all -l function=$FUNCTION
```

Comment:
this should be handled already by: https://github.com/markfisher/sk8s/blob/master/event-dispatcher/src/main/java/io/sk8s/event/dispatcher/EventDispatchingHandler.java#L66

Reply:
That's what @trisberg says. It wasn't working for me.  I'll take it out.